### PR TITLE
Add two solutions for Task4

### PR DESCRIPTION
**Pull request by vadimkashko**
I just lost it)

### CODE REVIEW

File: Tasks/Kashko/task_04/task_04_one_variable.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+print(
+    list(
+        filter(
+            None,
+            sum(
+                zip_longest(map(int.__mul__, numbers[::2], numbers[1::2]),
+                            map(int.__add__, numbers[1::2], numbers[2::2])),
+                ()))))
+
+print(sum(compress(numbers, map(partial(int.__rmod__, 2), numbers))))
```
Comment (vadimkashko) [inline Tasks/Kashko/task_04/task_04_one_variable.py:42]: I know it is wrong to call magic methods in straight)

File: Tasks/Kashko/task_04/task_04_one_variable.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    list(
+        filter(
+            None,
+            sum(
+                zip_longest(map(int.__mul__, numbers[::2], numbers[1::2]),
+                            map(int.__add__, numbers[1::2], numbers[2::2])),
+                ()))))
+
+# Calculate sum of all odd numbers
+print(sum(compress(numbers, map(partial(int.__rmod__, 2), numbers))))
```
Comment (vadimkashko) [inline Tasks/Kashko/task_04/task_04_one_variable.py:43]: I know that it's usually wrong to call magic methods in straight)

[the remark this answers is not in the document]
Comment (MikitaTsiarentsyeu) [reply Tasks/Kashko/task_04/task_04_one_variable.py:43]: Yep, it's not recommended usually, but for this task everything is acceptable)

**Review by MikitaTsiarentsyeu — APPROVED** [review]
This is SO inventive solution, I like it very much)) 100 point for the simple solution and another 50 goes to you bank for the additional task

**Pull request merged**